tabs/src/components/teamsappzeronotes.svg

```diff
@@ -0,0 +1,76 @@
+                        <g transform="translate(7.000000, 5.000000)">
+                            <path d="M16.9,9.5 L15.3,16 C14.9,17.4 16.6,18.3 17.5,18.6 L13.2,19.7 C9.7,20.4 6,19.4 3.2,17.1 L0.4,14.3 L0.4,14.3 C1.3,14.5 3.2,14.3 3.6,12.9 L5.2,6.4 L16.9,9.5 Z" id="Path" fill="url(#radialGradient-1)"></path>
+                            <g transform="translate(2.000000, 4.000000)" fill="#9B643D" id="Oval">
+                                <ellipse transform="translate(15.576760, 5.638180) rotate(-75.895668) translate(-15.576760, -5.638180) " cx="15.57676" cy="5.63818" rx="1.60007656" ry="1.50007177"></ellipse>
+                                <ellipse transform="translate(2.599580, 2.389740) rotate(-75.895668) translate(-2.599580, -2.389740) " cx="2.59958" cy="2.38974" rx="1.60007656" ry="1.50007177"></ellipse>
+                            </g>
+                            <path d="M9.5,14.5 L9.3,14.5 C5.7,13.6 3.6,10 4.5,6.4 L4.8,5.2 C5.7,1.6 9.3,-0.5 12.9,0.4 L13.1,0.4 C16.7,1.3 18.8,4.9 17.9,8.5 L17.6,9.7 C16.7,13.2 13.1,15.4 9.5,14.5 Z" id="Path" fill="#9B643D"></path>
+                        </g>
+                    </g>
+                </g>
+                <g id="Group" transform="translate(135.000000, 162.000000)">
+                    <rect id="Rectangle" fill="#F7C6C8" transform="translate(45.207000, 26.810660) rotate(-66.945109) translate(-45.207000, -26.810660) " x="25.0076609" y="6.61132085" width="40.3986783" height="40.3986783"></rect>
+                    <rect id="Rectangle" fill="#EE8F94" transform="translate(51.452025, 12.075735) rotate(-66.945109) translate(-51.452025, -12.075735) " x="47.3021608" y="-8.07360578" width="8.29972846" height="40.2986816"></rect>
+                    <rect id="Rectangle" fill="#F7C6C8" transform="translate(27.548520, 35.418540) rotate(-60.368829) translate(-27.548520, -35.418540) " x="7.34912601" y="15.219146" width="40.398788" height="40.398788"></rect>
+                    <rect id="Rectangle" fill="#EE8F94" transform="translate(35.505660, 21.427520) rotate(-60.368829) translate(-35.505660, -21.427520) " x="31.3557845" y="1.27812451" width="8.299751" height="40.298791"></rect>
+                </g>
+                <g id="Group" transform="translate(43.000000, 38.000000)">
+                    <path d="M173.3,121.9 L4.9,121.9 C2.3,121.9 0.2,119.8 0.2,117.2 L0.2,9.4 C0.2,6.8 2.3,4.7 4.9,4.7 L173.3,4.7 C175.9,4.7 178,6.8 178,9.4 L178,117.2 C178,119.8 175.9,121.9 173.3,121.9 Z" id="Path" fill="#6264A7"></path>
+                    <path d="M172.3,113 L172.3,13.7 C172.3,11.7 170.5,10.2 168.4,10.2 L88.7,10.2 L88.7,116.6 L168.4,116.6 C170.6,116.5 172.3,114.9 172.3,113 Z" id="Path" fill="#EAEAF3"></path>
+                    <path d="M10,10.1 C7.8,10.1 6.1,11.7 6.1,13.6 L6.1,113 C6.1,115 7.9,116.5 10,116.5 L88.6,116.5 L88.6,10.1 L10,10.1 Z" id="Path" fill="#FAF9F8"></path>

```

Comment:
I noticed theres a d attributer with random numbers in multiple tags, what are they for?

Reply:
.svg file *sigh*

I have publicly embarrassed myself. NVM.